src/docs/reference/modules/Resources/README.md

```diff
@@ -50,7 +50,7 @@ The `OrchardCore.Resources` module provides some commonly used ones:
 | jQuery-ui-i18n        | Script | 1.7.2         | jQuery-ui      |
 | jquery.easing         | Script | 1.4.1         | -              |
 | jquery-resizable-dom  | Script | 0.35.0        | -              |
-| js-cookie             | Script | 3.0.1         | jQuery         |
+| js-cookie             | Script | 3.0.5         | jQuery         |
```

Comment:
remove `jQuery` from the `Dependencies` column as it does not depend on jQuery.

Reply:
ok dependency to jQuery removed in docs.